### ISSUE TITLE
Number assigned to a new work too high

### ISSUE THREAD

**Issue by MarcinKrawczuk** [SChAth/ScAthiop#2694]
![Zrzut ekranu 2024-11-15 205507](https://github.com/user-attachments/assets/95ef3d40-edd1-43f9-a1c9-be97a7a4e979)

Sorry for posting on a Friday night but I was trying to create a work record and this happened, it looks wrong, the number is too high, it should be 7XXX, shouldn't it? 

@eu-genia

**Comment by eu-genia** [comment]
yes, this is a bug that keeps reemerging,  please rename your stub as `LIT7149Life `should be ok now

**Comment by thea-m** [comment]
Again with LIT14518PPMalkFan
Could you give me the correct ID for the renaming?

edit: also LIT14519PPAynaTe. I will stop creating records now as the issue is persisting :)

**Comment by eu-genia** [comment]
rename LIT7157PPMalkFan
and LIT7158PPAynaTe 

should be ok now

once a high number appears the "error" will of course persist, as each new number is the maximum old one + 1. so it is best to stop after the first strange number )

**Comment by thea-m** [comment]
thank you!

**Comment by thea-m** [comment]
LIT14518MGAbraset

**Comment by eu-genia** [comment]
I have deleted the record. To avoid confusion please create a new one, the numbering should be ok. Thank you

**Comment by karljonaskarlsson** [comment]
LIT14518PPSolomon

**Comment by eu-genia** [comment]
Removed, thank you! Please recreate the file with a new number

**Comment by karljonaskarlsson** [comment]
> Removed, thank you! Please recreate the file with a new number

Many thanks!
